atlassian_jwt_auth/contrib/tests/aiohttp/__init__.py

```diff
@@ -1,7 +1,17 @@
 import sys
 
+import nose
+
+
 if sys.version_info < (3, 5):
-    import nose
     raise nose.SkipTest('Skipping tests for Python version < 3.5')
+else:
+    try:
+        import asynctest
+    except ImportError:
+        import nose
```

Comment:
`nose` should be already imported at this point. See line [#3](https://github.com/atlassian/asap-authentication-python/pull/39/commits/3b798fcf3de00ccdb4d0d001c2dd1893376d5fde#diff-1c4e2b2b8233f7ad87117902ec438cefR3)